Samples-Typescript/Dashboard/PdfViewer/pdfviewerdialog.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <style>
+    .container-header {
+      position: relative;
+      display: block;
+    }
+  </style>
+  <head>
+    <title>Edit PDF URL</title>
```

Comment:
Making sure that the string `Edit PDF URL` is the correct one to use or is this a placeholder text that didn't get updated?

Reply:
It's not used in the actual dialog as the dialog title is set by Extensions code using the Extension name.  We discussed and decided to leave it.